src/ios/SimpleCameraPreview.m

```diff
@@ -27,6 +27,8 @@ - (void) enable:(CDVInvokedUrlCommand*)command {
     locationManager = [[CLLocationManager alloc] init];
     locationManager.delegate = self;
     locationManager.desiredAccuracy = kCLLocationAccuracyBest;
+    locationManager.distanceFilter = kCLHeadingFilterNone;
```

Comment:
https://developer.apple.com/documentation/corelocation/cllocationmanager/1423500-distancefilter

Discussion

This distance is measured relative to the previously delivered location. Use the value [kCLDistanceFilterNone](https://developer.apple.com/documentation/corelocation/kcldistancefilternone) to be notified of all movements. The default value of this property is kCLDistanceFilterNone.

This property is used only in conjunction with the standard location services and is not used when monitoring significant location changes.